src/Packetery/Module/Options/Page.php

```diff
@@ -582,13 +583,38 @@ public function validatePacketeryOptions( array $options ): array {
 		 */
 		$packeteryContainer = $form[ self::FORM_FIELDS_CONTAINER ];
 		$packeteryContainer->setValues( $options );
+
+		// Validation of hidden fields.
+		$weightEnabled = $packeteryContainer->getComponent( 'default_weight_enabled' );
+		if ( $weightEnabled && $weightEnabled->getValue() === false ) {
+			$defaultWeight = $packeteryContainer->getComponent( 'default_weight' );
+			if ( $defaultWeight ) {
+				$defaultWeight->addRule( Form::FLOAT )->addRule( Form::MIN, null, 0.1 );
+			}
+		}
+		$defaultDimensionsEnabled = $packeteryContainer->getComponent( 'default_dimensions_enabled' );
+		if ( $defaultDimensionsEnabled && $defaultDimensionsEnabled->getValue() === false ) {
```

Comment:
raději bych se podíval jestli to je instance Checkbox a pak to má být === true? A ne false?

Reply:
Komponentu `default_dimensions_enabled` vytváříme jako checkbox v definici formuláře, tak proč kontrolovat?
Kontrola na `false` je tam schválně, aby se dodatečná validace provedla jen pokud je formulář skrytý. Když je viditelný validace se provádí na základě `addConditionOn`.